content/community/courses.md

```diff
@@ -1,55 +1,56 @@
 
-- [Fullstack React](https://www.fullstackreact.com/) - The up-to-date, in-depth, complete guide to React and friends.
+- [Frontend Masters](https://frontendmasters.com/courses/)<span dir='rtl'> - React لائبریری اور دیگر Frontend فریم ورک کو سیکھنے کے لئے ویڈیو کورسز کی تربیت کرنا ۔</span>
 
-- [Pure React](https://daveceddia.com/pure-react/) - A step-by-step guide to mastering React.
+- [Fullstack React](https://www.fullstackreact.com/)<span dir='rtl'> - تازہ ترین ، گہرائی میں ، مکمل رہنمای کے ساتھ React دستیاب ھے ۔</span> دستیاب ھے ۔
 
-- [React for Beginners](https://reactforbeginners.com/) - Learn React in just a couple of afternoons.
+- [Pure React](https://daveceddia.com/pure-react/)<span dir="rtl"> - React میں مہارت حاصل کرنے کے لئے ایک قدم بہ قدم رہنما ۔</span>
 
-- [React for Designers](https://designcode.io/react) - A 6-hour React course for designers, by designers.
+- [React for Beginners](https://reactforbeginners.com/)<span dir="rtl"> - صرف چند دن میں React سیکھین ۔</span>
+ 
+- [React for Designers](https://designcode.io/react)<span dir="rtl"> - React کا 6 گھنٹے کا کورس ڈیزائنرز کے ذریعہ ، ڈیزائنرز کے لئے۔</span>
 
-- [React Essentials for Designers](https://learnreact.design) - React courses tailored for designers: the fundamentals, capabilities, limitations and how they relate to design.
+- [React Essentials for Designers](https://learnreact.design)<span dir='rtl'> - React
+ڈیزائنرز کے لئے تیار کردہ کورسز: بنیادی اصول ، قابلیت ، حدود اور ان کا ڈیزائن سے کس طرح تعلق ہے ۔</span>
+ 
+- [Essential React](https://learnreact.com/lessons/2018-essential-react-1-overview) - کوڈ لکھ کر سیکھنا پسند کرنے والوں کے لئے ایک تعلیمی نصاب ، یہ کورس انہیں جلدی سیکھنے اور جدید ترین ایپلیکیشنز بنانے کے قابل بنائے گا ۔
 
-- [Essential React](https://learnreact.com/lessons/2018-essential-react-1-overview) - A crash course for doers, moving fast from "Hello World" to advanced component composition.
 
-- [React Training: Advanced React.js](https://courses.reacttraining.com/p/advanced-react) - Take your React skills to the next level.
+- [React Training: Advanced React.js](https://courses.reacttraining.com/p/advanced-react)<span dir='rtl'> - React میں اپنی صلاحیتوں کو ترقی دیں ۔</span>
 
-- [Tyler McGinnis](https://tylermcginnis.com/courses) - Tyler McGinnis provides access to his courses for a monthly fee. Courses include "React Fundamentals" and "Universal React".
+- [Tyler McGinnis](https://tylermcginnis.com/courses)<span dir='rtl'> - Tyler McGinnis کے ساتھ سیکھیں ، جو ماہانہ تربیتی سیشن پیش کرتا ہے۔ ان کورسز میں بنیادی باتیں سیکھنا بھی  شامل ہے ۔</span>
 
-- [Mastering React](https://codewithmosh.com/p/mastering-react/) - Build professional interactive apps with React.
+- [Mastering React](https://codewithmosh.com/p/mastering-react/)<span dir='rtl'> - React لائبریری کے ساتھ انٹرایکٹو ایپلی کیشنز بنائیں ۔</span>
 
-- [React Tutorial](https://react-tutorial.app) - Learn React step by step in an interactive environment with flashcards.
```

Comment:
Why this line is removed? We need to re-add this

Reply:
Probably the outcome of a merge conflict resolution. Will revert it.